Paymetheus.Decred/Wallet/Checksum.cs

```diff
@@ -74,6 +75,26 @@ private static byte[] Hash(byte[] value)
 
             return hash;
         }
+
+        // Returned array contains the double BLAKE256 hash.
+        public static byte[] DoubleBlake256(byte[] value)
+        {
+            using (var hasher = new Blake256())
+            {
+                var intermediateHash = hasher.ComputeHash(value);
+                return hasher.ComputeHash(intermediateHash);
+            }
+        }
+
+        // Returned array contains the double SHA256 hash.
+        public static byte[] DoubleSha256(byte[] value)
```

Comment:
Since this is only called by the WalletSeed class, and only because the wallet seed checksum uses SHA256 instead of BLAKE256 like the rest of the decred tooling, I would prefer this to be a private method there.